test/qi/rule2.cpp

```diff
@@ -157,6 +157,32 @@ main()
         BOOST_TEST(s == "abcdef");
     }
 
+    {
+        typedef boost::fusion::vector<int, int> V;
+        typedef rule<char const*, V(), space_type> R;
+        V val;
+        R r;
+        r %= int_ >> int_;
```

Comment:
This is redundant https://github.com/boostorg/spirit/blob/95e1432c0c81bb107283ffaedc1d4c15f57ee747/test/qi/sequence.cpp#L120

Reply:
So, what you suggest for this PR? Close?!

Reply:
Remove the redundant test case.